app/controllers/task_controller.rb

```diff
@@ -1,13 +1,15 @@
 class TaskController < ApplicationController
```

Comment:
You want to follow the rails conventions and make this controller plural -- the PeopleController is sort of confusing in this regard